### ISSUE TITLE
Chat responses are not read

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#188835]
Testing #188537

- Type a request, press enter
- Nothing from the response is ever read

I think it's supposed to do this?

**Comment by meganrogge** [comment]
Are you in screen reader optimized mode?

**Comment by meganrogge** [comment]
also any errors in dev tools console?

**Comment by meganrogge** [comment]
here's the problem https://github.com/microsoft/vscode/blob/d0a179cf012f40fb50172a40ac9b7dba4f6431ba/src/vs/workbench/contrib/chat/browser/chatAccessibilityService.ts#L44

**Comment by meganrogge** [comment]
cc @rperez030 and @jooyoungseo this is the error you were encountering. should be fixed in insider's tomorrow

**Comment by meganrogge** [comment]
regression caused by the fix for https://github.com/microsoft/vscode/issues/188417
